src/services/gridData.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+
+import interceptor from "./utils/interceptor";
+
+interceptor.initializeInterceptor();
+
+export const getGridData = async (url) => {
+  try {
+    const response = await axios.get(url);
```

Comment:
we can create an axios instance so that it uses same interceptor every time without including interceptor. We can create a hook for axios and use that to make api calls

Reply:
hope i am clear with this

Reply:
@kuldep7  , yes , it is clear